### PR TITLE
Added a fix for setting Priority as required field 

### PR DESCRIPTION
**Pull request by Akasurde**
Fixes: https://fedorahosted.org/freeipa/ticket/6335

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

File: install/ui/src/freeipa/policy.js [hunk 1]
```diff
@@ -132,4 +135,4 @@ exp.register = function() {
 phases.on('registration', exp.register);
 
 return exp;
-});
\ No newline at end of file
+});
```
Comment (stlaz) [inline install/ui/src/freeipa/policy.js]: Please, turn off newlines at the end of file in your text editor and/or fix this. Other than that LGTM.

Comment (Akasurde) [inline install/ui/src/freeipa/policy.js]: OK

**Comment by stlaz** [comment]
Looks OK to me, thanks.

**Comment by Akasurde** [comment]
@stlaz Thanks

**Comment by mbasti-rh** [comment]
Sorry guys, but ticket is in closed milestone you need a new one.

**Comment by Akasurde** [comment]
@mbasti-rh I re-opened the ticket, Could you please provide ack ?

**Comment by mbasti-rh** [comment]
@Akasurde I said we need a *new* ticket for it. I opened one for you https://fedorahosted.org/freeipa/ticket/6335

**Comment by mbasti-rh** [comment]
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/8149b762b424d1ce7a26847386715ca98038b230

**Pull request closed without merging**